db/models.py

```diff
@@ -1 +1,32 @@
 from django.db import models
+
+
+class Race(models.Model):
+    RACE_CHOICE = [
+        ("Elf", "Elf"),
+        ("Dwarf", "Dwarf"),
+        ("Human", "Human"),
+        ("Ork", "Ork")
+    ]
+    name = models.CharField(max_length=255, unique=True, choices=RACE_CHOICE)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
```

Comment:
Consider using a more appropriate data type for the 'bonus' field if it represents a numeric value, such as IntegerField or FloatField.